packages/aside/index.ts

```diff
@@ -1,9 +1,12 @@
 import { App } from 'vue'
+import { SFCWithInstall } from '../utils/types'
```

Comment:
Also, use `import type { SFCWithInstall } from '@element-plus/utils/types'` instead.

Reply:
Done.
The packaged `lib` does not have the `el-utils` directory, the correct directory is `utils`., the correct directory is `utils`. It may be a problem with `gen-type.js`
![image](https://user-images.githubusercontent.com/25665423/102843014-48cfaf00-4443-11eb-875a-ccec06df4428.png)

Reply:
Aye, util was packaged independently. We need to fix that.

Reply:
I push a commit to fix `@element-plus/uitls` replacement. Should i push a sinlge PR for this? 
d2bc49644307a9eec3cabf572c1eb744a34ecaca